ios/ci_scripts/ci_post_clone.sh

```
@@ -1,15 +1,104 @@
+# Load nvm and install Node.js 18.17.0
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+
+nvm install 18.17.0
+nvm use 18.17.0
+nvm alias default 18.17.0
+
+# Verify Node.js version
+node -v
+
+sudo ln -sf $(which node) /Users/local/.nvm/versions/node/v18.17.0/bin/node
```

Shellcheck (0.9.0-1)

[warning] Quote this to prevent word splitting.

(SC2046)